ethstaker_deposit/intl/en/cli/existing_mnemonic.json

```
@@ -19,7 +19,7 @@
         "arg_validator_start_index": {
             "help": "Enter the index (key number) you wish to start generating more keys from. For example, if you've generated 4 keys in the past, you'd enter 4 here.",
             "prompt": "Enter the index (key number) you wish to start generating more keys from. For example, if you've generated 4 keys in the past, you'd enter 4 here.",
-            "confirm": "Please repeat the index to confirm"
+            "confirm": "Please repeat the index (key number) to confirm"
```

I understand this is directly from the other PR but I'm not a fan of the change.
Maybe we should be more explicit?
```suggestion
            "confirm": "Please repeat the validator start index to confirm"
```